src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -183,7 +183,10 @@ export async function getContainerLogList(logListUrl) {
 export async function getContainerLog(tailLogUrls, fullLogUrls, logType) {
   const res = await fetch(tailLogUrls[logType]);
   if (!res.ok) {
-    throw new Error(res.status);
+    if (String(res.status) === '403') {
```

Comment:
Why convert it to string? Would `res.status === 403` work as expected?

Reply:
No, I have to add `String()` so it can work as expected. `res.status == '403'` work too but it can't pass the lint check.